config/locales/en.yml

```diff
@@ -324,7 +320,7 @@ en:
     preparing_4g_wireless_routers:
       title: Preparing 4G routers
       description: Find out how 4G wireless routers are secured, how much data users get, how long the contract is and records you should keep.
-      audience: responsible_body_users
+      audience: to_be_removed
```

Comment:
can this be removed?

Reply:
Naming is hard ;)

The next ticket I'm picking up refactors how we build the device guidance index and the guide pages themselves. THis was the quickest way I could remove it in this PR but it will be removed in the next ticket.